### PR TITLE
Quadrat: Add table styles

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Adds table styles for quadrat to match EDoXTvsZt4lMHItzOczFQw-fi-50%3A767

Frontend:
Screenshot 2021-04-28 at 11 45 53

Editor:
Screenshot 2021-04-28 at 11 46 32

#### Related issue(s):
Part of #3640

### CODE REVIEW

**Review by jffng — COMMENTED** [review]
It looks like some of the border-width CSS is being mis-applied. Here is what I get using the [theamdemo table content](pcrugM-gx-p2): 

Screen Shot 2021-04-28 at 4 19 05 PM

**Comment by scruffian** [comment]
Thanks, I added a commit to fix it.

File: quadrat/sass/blocks/_table.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.wp-block-table {
+	th {
+		font-weight: 500;
+		text-align: left;
```
Comment (jffng) [inline quadrat/sass/blocks/_table.scss]: This interferes with Gutenberg's text align utility classes: 

Screen Shot 2021-04-29 at 12 18 10 PM

Could we remove it? Users still have the option of aligning the header to the left.

**Review by jffng — COMMENTED** [review]
I left one comment, other than that this LGTM!

File: quadrat/sass/blocks/_table.scss (hunk 1 above)
Comment (scruffian) [reply quadrat/sass/blocks/_table.scss]: I guess that's a question for @beafialho and @kjellr

Comment (kjellr) [reply quadrat/sass/blocks/_table.scss]: It's fine to remove this rule.

**Pull request merged**